aten/src/ATen/detail/XPUHooksInterface.h

```diff
@@ -55,6 +51,10 @@ struct TORCH_API XPUHooksInterface {
     return 0;
   }
 
+  virtual DeviceIndex current_device() const {
+    return -1;
```

Comment:
I know the value here is copy pasted from the cuda code but why is the current device -1 when XPU is not available? Why wouldn't that be an error like every other default impl here?

Reply:
Done. Raising an error looks better.